books/free-programming-books-bl.md

```diff
@@ -1,26 +1,119 @@
 ### Index
 
+* [Android](#android)
+* [Bootstrap](#bootstrap)
 * [C](#c)
+* [C++](#c-plus-plus)
 * [Competitive Programming](#competitive-programming)
-* [JavaScript](#JavaScript)
+* [C Sharp](#c-sharp)
```

Comment:
Let me see if I can get the linter to check the alphabetization of this list. C Sharp will come between C and C++ - take a look at the english lists.

Reply:
I've replaced c# between C and C++. Would you kindly check that out?